tests/conftest.py

```
@@ -0,0 +1,82 @@
+        importlib.reload(real_runtime)
+
+
+@pytest.fixture(autouse=True)
+def ensure_default_event_loop():
+    """Ensure legacy tests using get_event_loop() have a usable loop."""
+    try:
+        asyncio.get_running_loop()
+    except RuntimeError:
+        asyncio.set_event_loop(asyncio.new_event_loop())
+
+    yield
```

Ruff (0.16.1)

[warning] No teardown in fixture `ensure_default_event_loop`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)